reviews/incubation-in-toto.md

```diff
@@ -0,0 +1,194 @@
+  * Improve upstream release packaging for Debian and Arch Linux ([#279](https://github.com/in-toto/in-toto/pull/279), [#290](https://github.com/in-toto/in-toto/pull/290))
+
+More changes and small improvements are mentioned in the current release
+changelog.
+
+Further, the [Go implementation](https://github.com/in-toto/in-toto-golang) has had one
+pre-release, [v0.1.0](https://github.com/in-toto/in-toto-golang/releases).
+
+Beyond the current release other improvements to the broader reference
+implementation have been achieved.
+
+* A [formalized governance
+policy](https://github.com/in-toto/in-toto/blob/develop/GOVERNANCE.md) has been
+instituted project-wide. This includes not only the in-toto python reference
+implementation, but the specifications, implementations in other languages and
+cloud-native tooling.
+
+## Incubating Stage Criteria
+
+In addition to sandbox requirements, a project must meet the following
+criteria to become an incubation-stage project:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgment, are of adequate quality and scope
+
+  * We document adopters on
+    [our website](https://in-toto.io/integrations.html).
+  * The in-toto specification has inspired
+    the development of [Argos Notary](https://www.argosnotary.com/docs/00_overview/10_overview)
+    by Rabobank.
+  * Boxboat is currently [integrating](https://github.com/boxboat/in-toto-golang)
+    SPIFFE, a CNCF project, with in-toto. This will be presented at
+    [Cloud Native Security Day 2021](https://cnsecuritydayeu21.sched.com/event/iRn2).
+  * in-toto is also used by Cloud Native Application Bundles (CNAB), another
+    CNCF project, in [Signy](https://github.com/cnabio/signy).
+
+### Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project
+
+  * Maintainers of the project are listed in our
+    [MAINTAINERS.txt](https://github.com/in-toto/in-toto/blob/develop/MAINTAINERS.txt)
+    file. There are currently five core maintainers from Purdue University, New
+    York University, and Conda who cut releases.
+
+  * Several other people have commit access, but cannot craft releases. They are:
+    * Holger Levsen (Debian)
+    * Ofek Lev (Datadog)
+
+  * We have had contributions from people associated with different organizations.
+    Some of them, listed in no particular order:
+    * Trishank Karthik Kuppusamy (Datadog)
+    * Joshua Lock (VMWare)
+    * Dan Lorenc (Google)
+    * Christian Rebischke (Arch Linux)
+
+  * Maintainers are added and removed from the project as per the policies
+    outlined in the project [GOVERNANCE.md](https://github.com/in-toto/in-toto/blob/develop/GOVERNANCE.md)
+    file.
+
+  * Finally, in-toto participated in Google Summer of Code (GSOC) 2020 through the CNCF.
```

Comment:
Do we want to mention the CNCF blog article about the in-toto GSOC internship here?
https://www.cncf.io/blog/2020/10/07/gsoc-spotlight-my-google-summer-of-code-experience-at-cncf-in-2020/

It's something that underlines the welcome culture in the in-toto project imho :)

Reply:
this can be resolved.